console/console-init/ui/src/modules/connection-detail/ConnectionDetailPage.tsx

```diff
@@ -25,7 +25,7 @@ import { getFilteredValue } from "components/common/ConnectionListFormatter";
 import { IConnectionDetailResponse } from "types/ResponseTypes";
 import { Link } from "react-router-dom";
 import { RETURN_CONNECTION_DETAIL } from "graphql-module/queries";
-import { ConnectionLinksWithFilterAndPaginationPage } from "./ConnectionLinksWithFilterAndPaginationPage";
+import { ConnectionDetailToolbar } from "./components/ConnectionDetailToolbar/ConnectionDetailToolbar";
```

Comment:
path should be "./components/ConnectionDetailToolabr"

Reply:
Agreed, like I said, these paths got automatically updated, I will be changing all the paths like these